### DISCUSSION TITLE
What is the smartest creature on earth?

### DISCUSSION THREAD

**Discussion by opticaldrive** [ParanoidUser/yolo#1038] [Galaxy Trivia] [answered]
what is the smartest creature on earth?

**Comment by peterbaccaro** [comment, marked as answer]
**Humans** are widely considered the most intelligent species on Earth due to their advanced cognitive abilities, problem solving skills, language and capacity for complex reasoning.

**Great Apes** (Chimpanzees, Bonobos, Gorillas, Orangutans) are considered the most intelligent animals. They share a significant amount of DNA with humans and display high levels of problem-solving ability, tool usage and social intelligence.

**Comment by ParanoidUser** [comment]
Great question! Keep them coming 🙈🙉🙊
